assert/array_empty.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TEMP FUNCTION
```

Comment:
Would it be a big pain to put this `assert` dir inside `tests`? It seems an odd fit for the top level.

Reply:
i bet i could work that out

Reply:
fixed